docs/menu.cue

```diff
@@ -89,6 +89,8 @@
 #AccordionMenuItem: {
 	kind: "accordion"
 	text: string
+	// If the accordion is open by default.
+	defaultExpanded?: bool
```

Comment:
I'd make this a bool with a default value of false, instead of an optional bool:

```suggestion
	defaultExpanded: bool | *false
```